docs/EVG.md

```diff
@@ -319,7 +319,9 @@ Each CA must develop, implement, enforce, display prominently on its Web site, a
 
 ###  8.2.2.  Disclosure
 
-Each CA must publicly disclose their EV Policies through an appropriate and readily accessible online means that is available on a 24x7 basis.  The CA is also REQUIRED to publicly disclose its CA business practices as required by WebTrust for CAs and ETSI TS 102 042 and ETSI EN 319 411-1.  The disclosures MUST be structured in accordance with either RFC 2527 or RFC 3647.
+Each CA must publicly disclose its Certificate Policy and/or Certification Practice Statement through an appropriate and readily accessible online means that is available on a 24x7 basis. The CA SHALL publicly disclose its CA business practices to the extent required by the CA's selected audit scheme (see Section 17.1).
```

Comment:
This aligns with the BRs approach to referencing the audit criteria, and makes sure the disclosures match the selected audit criteria.